memdx/durability.go

```diff
@@ -0,0 +1,20 @@
+package memdx
+
+// DurabilityLevel specifies the level to use for enhanced durability requirements.
+type DurabilityLevel uint8
+
+// TODO(chvck): Should this start
+const (
+	// DurabilityLevelNone indicates that no durability is needed.
+	DurabilityLevelNone = DurabilityLevel(0)
```

Comment:
I think we previously ran into issues with not being able to detect the difference between 'Unspecified' and 'None'.  Perhaps we should make 0 be undefined instead (I think it's acceptable to need to map values for the purposes of gocb compatibility)?